Resources/views/Section/not_found.html.twig

```diff
@@ -1,14 +0,0 @@
-{% extends "eZPlatformUIBundle::pjax.html.twig" %}
-
-{% trans_default_domain "section" %}
-
-{% block title %}{{ 'section.not_found.title'|trans }}{% endblock %}
```

Comment:
these translation keys (title and message) could be removed from: `Resources/translations/section.en.xlf`